spec.html

```diff
@@ -29918,68 +29930,68 @@ <h1>Runtime Semantics: GetSubstitution ( _matched_, _str_, _position_, _captures
               </tr>
               <tr>
                 <td>
-                  0x0024, N
-                  <br>
-                  Where
-                  <br>
-                  0x0031 &le; N &le; 0x0039
+                  0x0024, 0x0030
                 </td>
                 <td>
-                  `$n` where
-                  <br>
-                  `n` is one of `1 2 3 4 5 6 7 8 9` and `$n` is not followed by a decimal digit
+                  `$0`
                 </td>
                 <td>
-                  The _n_<sup>th</sup> element of _captures_, where _n_ is a single digit in the range 1 to 9. If _n_ &le; _m_ and the _n_<sup>th</sup> element of _captures_ is *undefined*, use the empty String instead. If _n_ &gt; _m_, no replacement is done.
+                  `$0`
                 </td>
               </tr>
               <tr>
                 <td>
-                  0x0024, N, N
-                  <br>
-                  Where
+                  0x0024, N
                   <br>
-                  0x0030 &le; N &le; 0x0039
+                  where 0x0031 &le; N &le; 0x0039
                 </td>
                 <td>
-                  `$nn` where
-                  <br>
-                  `n` is one of `0 1 2 3 4 5 6 7 8 9`
+                  `$`_n_
+                  where _n_ is an expansion of |NonZeroDigit|
                 </td>
                 <td>
-                  The _nn_<sup>th</sup> element of _captures_, where _nn_ is a two-digit decimal number in the range 01 to 99. If _nn_ &le; _m_ and the _nn_<sup>th</sup> element of _captures_ is *undefined*, use the empty String instead. If _nn_ is 00 or _nn_ &gt; _m_, no replacement is done.
+                  If the MV of _n_ &gt; _m_, the replacement is the matched code units (equivalent to no replacement).
+                  <br>
+                  Otherwise, if the element in _captures_ at index equal to the MV of _n_ minus 1 is *undefined*, the replacement is the empty String.
+                  <br>
+                  Otherwise, the replacement is the element in _captures_ at index equal to the MV of _n_ minus 1.
```

Comment:
I considered \<emu-alg>, but ultimately preferred the appearance without it: https://deploy-preview-1732--ecma262-snapshots.netlify.com/#table-45 (and I'd get rid of the \<emu-alg> for `$<…>` if I could find a good way to do so). As always, though, I'm willing to change if there's consensus.

As for _N_ vs. _n_, we must use the latter because the former is not defined as the expansion of a lexical grammar production and thus has no MV.

Reply:
> As for _N_ vs. _n_, we must use the latter because the former is not defined as the expansion of a lexical grammar production and thus has no MV.

Well, of course, you'd need to change the name in column 2 as well. Sorry that wasn't clear. Anyhow,  the point there was to free up the name `_n_` for `the MV of _N_`, but that isn't needed if you don't convert to `<emu-alg>`.